prow/plugins/heart/heart.go

```diff
@@ -51,6 +51,9 @@ func init() {
 
 func helpProvider(config *plugins.Configuration, enabledRepos []string) (*pluginhelp.PluginHelp, error) {
 	// The {WhoCanUse, Usage, Examples} fields are omitted because this plugin is not triggered with commands.
+	if config.Heart == nil {
```

Comment:
Instead of doing these checks at run-time, check that the plugins that are enabled are configured at config load time.

Reply:
Alright ! Should i create a function that validate the rest of `Configuration` fields like `validatePlugins`?